set.mm

```diff
@@ -632273,6 +632273,103 @@ fixed reference functional determined by this vector (corresponding to
       TUICVSAUJUTUKPUQVQVTVRVALPUQVLWAVNVLDKCWAULADUMUNPUTVAURUOUPS $.
   $}
 
+  ${
+    $d A k $.  $d M k $.  $d N k $.  $d A j k $.  $d K j $.  $d M j k $.
+    $d N j k $.  $d A k x $.  $d K x $.  $d M k x $.  $d N k x $.  $d M x y $.
+    $d N x y $.  $d M k x y $.  $d N k x y $.  $d K j k $.  $d K k x $.
+    $d A k x y $.
+    $( Product split into two factors.  (Contributed by metakunt,
+       20-Apr-2024.) $)
+    prodsplit $p |- ( ( ( ( M e. ZZ /\ N e. ZZ /\ M <_ N ) /\ A e. CC )
+      /\ K e. NN ) ->
+     ( prod_ k e. ( M ... N ) A x. prod_ k e. ( ( N + 1 ) ... ( N + K ) ) A )
+                         = prod_ k e. ( M ... ( N + K ) ) A ) $=
```

Comment:
Nice! 
I think it's not always evident to use inductions theorems, so that's a very good step!
Here my remark would be that in the case of this theorem all sets are finite, so `fprodsplit` should apply.

Reply:
Wait if i do that, what would the hypotheses look like? I'd love to have a more general version of it. At the bottom I have drafted an ansatz of my new proof strategy. How would I replace my hypotheses to fit general sets.